compendium/dnd5e.spells.json

```
@@ -1,52 +1,92 @@
 {
     "entries": {
         "Acid Arrow": {
-            "name": ""
+            "name": "Melfs Säurepfeil",
+            "description": "<p>Ein schimmernder grüner Pfeil schießt auf ein Ziel in Reichweite zu und explodiert in einem Sprühregen aus Säure. Führe eine Fernkampf-Zauberangriff gegen das Ziel aus. Bei einem Treffer erleidet es sofort 4W4 Säureschaden und 2W4 Säureschaden am Ende seines nächsten Zuges. Verfehlst du, spritzt genug Säure auf das Ziel, um die Hälfte des anfänglichen Schadens zu verursachen, ohne jedoch Schaden am Ende des nächsten Zuges zuzufügen.<br /><strong>Auf höheren Graden:</strong> Wenn du diesen Zauber mit einem Zauberplatz des 3. oder höheren Grades wirkst, steigt der Schaden (sowohl der anfängliche als auch der spätere) für jeden Grad über den 2. hinaus um 1W4.</p>",
+            "material": "ein zerstoßenes Rhabarberblatt und der Magen der Natter",
+            "source": "Phb. p. 249"
```

```suggestion
            "source": "Phb. p. 249"
```

In anderen Dateien benutzen wir SHB statt PHB

---

PHB werden wir noch gegen SHB ersetzen.

---

#done